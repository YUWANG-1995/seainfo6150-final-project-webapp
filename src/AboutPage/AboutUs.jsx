import React from "react";
import logo1 from "../images/logos/aboutusLogo.png";
import logo2 from "../images/logos/logo2.png";
import logo3 from "../images/logos/logo3.jpeg";
import pic from "../images/AboutusPage/customer.png";
import styles from "./AboutUs.module.css";

const AboutUs = () => {
  return (
    <div className={styles.container}>
      <h1>About Us</h1>
      <div>
      <picture>
         <source media="(max-width: 600px)"
            srcSet={logo3}
         />
         <source media="(max-width: 1200px)"
            srcSet={logo2}
         />
         <img className={styles.tupian} src={logo1} alt="sports car" />
      </picture>

      </div>
      <div className={styles.container}>
        <h3>Our History</h3>
        <p>we are a new car dealer company, we start from 2020, our groups
         are consist of all young people, we have a team of experts ready to provide
         you a best service to make you have a memorable experience.
        </p>
      </div>
      <div className={styles.serviceDisplay}>
        <div>
          <img className={styles.tupian} src={pic} alt="customer service tupian" />
        </div>
        <div className={styles.container}>
          <h3>Our Service</h3>
          <p>we provide 7x24 hours service, if you meet any problems, please contact me immediately,
             we will provide professional and timely service.
             we also provide 1 vs 1 personal assistant, our assistant will help you solve any problems,
             and we will collect your feedback every year.
          </p>
        </div>
      </div>
     </div>
  )
}

export default AboutUs;
