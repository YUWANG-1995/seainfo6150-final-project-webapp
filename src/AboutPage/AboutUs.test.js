import React from "react";
import { render } from "@testing-library/react";
import AboutUs from "./AboutUs.jsx";

it("renders the AboutUs Component Correctly", () => {
  const { container } = render(<AboutUs />);
  expect(container).toMatchSnapshot();
});
