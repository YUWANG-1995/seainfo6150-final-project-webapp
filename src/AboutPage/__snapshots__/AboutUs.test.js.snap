// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the AboutUs Component Correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      About Us
    </h1>
    <div>
      <picture>
        <source
          media="(max-width: 600px)"
          srcset="logo3.jpeg"
        />
        <source
          media="(max-width: 1200px)"
          srcset="logo2.png"
        />
        <img
          alt="sports car"
          class="tupian"
          src="aboutusLogo.png"
        />
      </picture>
    </div>
    <div
      class="container"
    >
      <h3>
        Our History
      </h3>
      <p>
        we are a new car dealer company, we start from 2020, our groups are consist of all young people, we have a team of experts ready to provide you a best service to make you have a memorable experience.
      </p>
    </div>
    <div
      class="serviceDisplay"
    >
      <div>
        <img
          alt="customer service tupian"
          class="tupian"
          src="customer.png"
        />
      </div>
      <div
        class="container"
      >
        <h3>
          Our Service
        </h3>
        <p>
          we provide 7x24 hours service, if you meet any problems, please contact me immediately, we will provide professional and timely service. we also provide 1 vs 1 personal assistant, our assistant will help you solve any problems, and we will collect your feedback every year.
        </p>
      </div>
    </div>
  </div>
</div>
`;
